.dep-versions

```diff
@@ -8,7 +8,7 @@ enzyme=v0.0.149
 
 # For a custom PL version, update the package version here and at
 # 'doc/requirements.txt
-pennylane=0.42.0-dev19
+pennylane=0.42.0-dev22
```

Comment:
Do you still need to update this dev version after merging your second PR in PL?

Reply:
Done in [1d29b6f](https://github.com/PennyLaneAI/catalyst/pull/1619/commits/1d29b6fb0fc1fe1fe23b2e364fc920141d30cb26)